tools/update_plotlyjs.R

```diff
@@ -65,10 +62,29 @@ withr::with_dir(tmpdir, {
         file.path(lib_dir, "locales", sub("^plotly-locale-", "", basename(locales))),
         overwrite = TRUE
       )
-      # update plot schema
+      # update plot schema and (partial) bundles
       Schema <- jsonlite::fromJSON(Sys.glob("dist/plot-schema.json"))
+      tmp_file <- tempfile(pattern = "plotly_constants_", fileext = ".js")
+      utils::download.file(
+        url = paste0("https://raw.githubusercontent.com/plotly/plotly.js/", basename(zip), "/tasks/util/constants.js"),
+        destfile = tmp_file,
+        quiet = TRUE,
+        mode = "wb"
+      )
+      bundleTraceMap <-
+        paste0(readLines(tmp_file), collapse = "\n") |>
+        stringr::str_extract(pattern = "(?<=var partialBundleTraces = )\\{[^}]+\\}") |>
+        yaml::read_yaml(text = _)
```

Comment:
It'd be nice if this could also handle `const` as well as throw informative errors if the `str_extract()` comes up empty

Reply:
Addressed in https://github.com/plotly/plotly.R/pull/2336/commits/74a4f4f07159000851a75c68180c0ca313bc511a.